admin/admin.php

```diff
@@ -43,7 +43,8 @@ public function add_privacy_policy() {
 	 */
 	public function admin_menu() {
 
-		$this->settings_page_id = add_options_page( __( 'Password Protected', 'password-protected' ), __( 'Password Protected', 'password-protected' ), 'manage_options', 'password-protected', array( $this, 'settings_page' ) );
+		$capability = apply_filters( 'password-protected-options-page-capability', 'manage_options' );
```

Comment:
This is good, but will change the filter name to use underscores to match other filters:
`password_protected_options_page_capability`